CHANGELOG.rst

```diff
@@ -2,19 +2,40 @@
 Change history
 ==============
 
-2.x.x
+2.4.0
 =====
-*TBD*
+*November, 22, 2024*
 
 **New features**
 
-* Updated OAF version to 0.9.0. This upgrade allows admin users managing their sessions through the admin.
 * [#197] Added a ``migrate_to_v2`` management command which allows users of version ``1.0.0`` to migrate to version ``2.4.0``
   More information can be found in the `documentation <https://open-klant.readthedocs.io/en/latest/installation/migration.html>`
+* [#246] Add is_standaard_adres for DigitaalAdres
```

Comment:
```suggestion
* [#246] Add ``isStandaardAdres`` for ``DigitaalAdres``
```
You use camel case in all other descriptions for the fields, so let's make it consistent